modules/ossm-rn-new-features.adoc

```diff
@@ -168,6 +190,28 @@ spec:
 * {SMProductShortName} on ARM64 architecture is not supported.
 * OpenTelemetry API remains a Technology Preview feature.
 
+== New features {SMProductName} version 2.3.6
+
+This release of {SMProductName} addresses Common Vulnerabilities and Exposures (CVEs), contains bug fixes, and is supported on {product-title} 4.10 and later versions.
+
+=== Component versions included in {SMProductName} version 2.3.6
+
+|===
+|Component |Version
+
+|Istio
+|1.14.6
```

Comment:
oc rsh -n istio-system23 istio-egressgateway-7d7cb9d55b-4gcl7 env | grep ISTIO_VERSION 
ISTIO_VERSION=1.14.5

It is 1.14.5